tasks/HelpMeCarry.tex

```diff
@@ -0,0 +1,76 @@
+\section{Help Me Carry [Party Host]}
+The robot helps a guest to carry some luggage to his/her car.
+
+\subsection{Main Goal}
+Robot helps a guest to carry a bag to the car which is parked outside and comes back home.
+
+\subsection{Focus}
+It includes understanding pointing gesture, object identification and manipulation, navigation in known and unknown environments, person-following while carrying a luggage, and social navigation.
+
+\noindent\textbf{Reward:} 500 pts for carrying the bag to the car's location and coming back home.
+
+\subsection{Bonus rewards}
+\begin{enumerate}[nosep]
+	\item Find the bag by understanding the pointing behaviour (100 pts).
+	\item Avoid all the obstacles outdoor (400 pts).
+\end{enumerate}
+
+
+\subsection{Setup}
+\begin{itemize}
+	\item \textbf{Location}: The test takes place inside and outside the arena.
+
+	\item \textbf{Start Location}: The robot starts at a predefined location in the living room.
+
+	\item \textbf{Guest}: There is only one guest in the house, standing in front of the robot at the start.
+	
+	\item \textbf{Bags' Location:} There are at least two bags placed between one to two meters  away from the guest's start position and both should be visible to the robot.
+	
+	\item \textbf{Pointing to the Bag:} When the test starts, the guest points to his/her bag and requests the robot to carry it to the car.
```

Comment:
- ~his/her~ → a
- ~guest~ → operator
- ~carry~ → take

Reply:
I think carry sounds more meaningful. You can take something out by using a trolley or by pulling it.